plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -714,6 +714,9 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
     }
 
     private fun startDragging(annotation: T): Boolean {
+      if (!annotation.isDraggable) {
```

Comment:
@Chaoba the only place where `startDragging` is called from is `onMoveBegin`, which also runs `queryMapForFeatures` before. Could we check `isDraggable` before and skip the query render features call then? Will save couple thread switches

Reply:
E.g. [here ](https://github.com/mapbox/mapbox-maps-android/blob/4a633d6c118424ffa852dd6fde158ab48ddbb10b/plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt#L647)

Reply:
Only after `queryMapForFeatures` can we get the annotation and know whether it's draggable or not.

Reply:
Oh, indeed, somehow missed the obvious part